core/io/ip_address.h

```diff
@@ -36,20 +36,19 @@
 struct IP_Address {
 private:
 	union {
-		uint8_t field8[16];
+		uint8_t field8[16]{};
```

Comment:
That looks a bit weird but we can only initialize one member from a union, and `clang-tidy` picked the first.